packages/core/getBusyTimes.ts

```diff
@@ -39,12 +50,22 @@ export async function getBusyTimes(params: {
         startTime: true,
         endTime: true,
         title: true,
+        eventType: {
+          select: {
+            afterEventBuffer: true,
+            beforeEventBuffer: true,
+          },
+        },
       },
     })
     .then((bookings) =>
-      bookings.map(({ startTime, endTime, title, id }) => ({
-        end: endTime,
-        start: startTime,
+      bookings.map(({ startTime, endTime, title, id, eventType }) => ({
+        start: dayjs(startTime)
+          .subtract((eventType?.beforeEventBuffer || 0) + (afterEventBuffer || 0), "minute")
```

Comment:
Thanks @CarinaWolli  for this fix :) 

Always subtract the beforeEventBuffer for this specific booking eventType - then add afterEventBuffer for the buffer you are trying to book. 

Preventing different event types from having a clash